stingray/gti.py

```diff
@@ -954,6 +968,86 @@ def join_equal_gti_boundaries(gti, threshold=0.0):
     return np.asarray(ng)
 
 
+def merge_gtis(gti_list, gti_treatment):
+    """Merge a list of GTIs using the specified method.
+
+    Invalid GTI lists (None or empty) are ignored.
+
+    Parameters
+    ----------
+    gti_list : list of 2-d float arrays
+        List of GTIs.
+
+    Other parameters
+    ----------------
+    gti_treatment : {"intersection", "union", "append", "infer", "none"}
+        Method to use to merge the GTIs. If "intersection", the GTIs are merged
+        using the intersection of the GTIs. If "union", the GTIs are merged
+        using the union of the GTIs. If "none", a single GTI with the minimum and
+        the maximum time stamps of all GTIs is returned. If "infer", the strategy
+        is decided based on the GTIs. If there are no overlaps, "union" is used,
+        otherwise "intersection" is used. If "append", the GTIs are simply appended
```

Comment:
Why do we need "append" strategy? 
We can always use "union". If they are mutually exclusive "union" == "append", if they are not "append" can't be used, right?